xdcs-agent/src/main/python/xdcs/conf/xdcs-agent.toml

```diff
@@ -5,13 +5,14 @@ fs_repo_path = "./data"
 send_path_executables = true
 
 [server]
-host = "xdcs-server"
-port.grpc = 8081
-port.ssh = 8082
+host = "127.0.0.1"
+port.grpc = 32081
+port.ssh = 32082
+allow_unknown_hosts = true
 
 [server.auth]
-name = "admin"
-password = "admin"
+name = "agent-1"
+key = "/etc/xdcs/dev/key"
```

Comment:
dev? przy instalacji? raczej proponuję `/etc/xdcs/agent-key` ale jak już instalujesz klucz prywatny to na pewno trzeba ustawić poprawne uprawnienia